hooks/pre-command

```diff
@@ -46,12 +67,20 @@ $(aws ecr get-login --no-include-email)
 docker_file="${BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_DOCKERFILE:-Dockerfile}"
 image_name="$(get_ecr_url ${BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_ECR_NAME})"
 
+build_args=""
+
+# read all arguments
+while read -r arg ; do
+  [[ -n "${arg:-}" ]] && build_args+=("--build-arg" "${arg}")
```

Comment:
does this work? is it not meant to be build_args+="--build-arg ${arg}" for string concatenation